crates/transaction-pool/src/validate/eth.rs

```diff
@@ -126,6 +126,8 @@ where
     kzg_settings: Arc<KzgSettings>,
     /// Marker for the transaction type
     _marker: PhantomData<T>,
+    ///
+    no_locals: bool,
```

Comment:
we also need this in the builder struct+setter function

Reply:
> we also need this in the builder struct+setter function

oh yeah, so for both builder + inner right?

Reply:
exactly